File: api/poptracker1/app/Models/Business.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Business extends Model
{
    use HasFactory;

    protected $fillable = [
        'name',
        'address',
        'coordinates',
        'population_id'
    ];

    public $timestamps = false;

    public function population(){
        return $this->belongsTo(Population::class, 'population_id', 'id');
    }

    public function populationHistory(){
        return $this->hasMany(PopulationHistory::class);
    }

}

File: app/poptrack/lib/data/models/business.dart
class Business{
  int? id;
  String name;
  String address;
  String coord;
  int? populationId;

  Business({this.id, required this.name, required this.coord, required this.address, this.populationId});

  factory Business.fromJson(Map<String, dynamic> json) {
    return Business(
      id: json['id'] as int,
      name: json['name'] as String,
      address: json['address'] as String,
      coord: json['coordinates'] as String,
      populationId: json['population_id'] as int
    );
  }

}